tools/reviewer/job/tasks/submitter.proto

```diff
@@ -1,12 +0,0 @@
-syntax = "proto3";
```

Comment:
Why do you remove this file?

Reply:
I didn't actively remove it.
Must be because one of my merges from `master`.

Reply:
Nevermind, it was still present in `master`, so I copied it to `local`.